compilers/concrete-optimizer/concrete-optimizer/src/optimization/dag/multi_parameters/analyze.rs

```diff
@@ -207,14 +206,13 @@ impl VariancedDag {
                     nb_partitions,
                     operator.partition().instruction_partition,
                 ),
-                Operator::LevelledOp { manp, .. } => {
-                    let max_var = operator
-                        .get_inputs_iter()
-                        .map(|a| a.variance()[operator.partition().instruction_partition].clone())
-                        .reduce(max_var)
-                        .unwrap();
-                    max_var.after_levelled_op(*manp)
-                }
+                Operator::LevelledOp { weights, .. } => operator
```

Comment:
extracting inputs in the pattern is clearer

Reply:
`inputs` in the `LevelledOp` is a `Vec<OperatorIndex>`, while `get_inputs_iter` returns an iterator over `VariancedDagOperator` in this context, which is different.